Sources/Testing/Testing.docc/EnablingAndDisabling.md

```diff
@@ -82,8 +82,14 @@ func isCold() async throws { ... }
 ```
 
 If a test is disabled because of a problem for which there is a corresponding
-bug report, you can use the ``Trait/bug(_:_:)-2u8j9`` or
-``Trait/bug(_:_:)-7mo2w`` function to show the relationship:
+bug report, you can use one of these functions to show the relationship
+between the test and the bug report:
+
+- ``Trait/bug(_:_:)``
+- ``Trait/bug(_:id:_:)-10yf5``
+- ``Trait/bug(_:id:_:)-3vtpl``
+
+For example, the following test cannot run due to bug number `"12345"`:
```

Comment:
The `.bug("12345")` a few lines below this should be `.bug(id: "12345")` now, I think